scripts/ssh_wrapper.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
Possibly the MANIFEST.in file needs adjustment to assure this file also gets distributed when making new releases through setuptools.